recipes/xlrd/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.0.0" %}
+{% set name = "xlrd" %}
+{% set sha256 = "0ff87dd5d50425084f7219cb6f86bb3eb5aa29063f53d50bf270ed007e941069" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+
+about:
+  home: 'http://www.python-excel.org/'
+  license: "BSD-style"
```

Comment:
> I did a patch of the license file and added it in the meta.yaml section accordingly.

That is fine but you could add the path to the license in the source like `license_file: docs/license.rst`

>  I added a second licnse tagger "BSD with advertising"

Thanks! You can also use `BSD 4-Clause` to be consistent with the rest of conda-forge licenses and make it easier to recognize, but that is optional.